### ISSUE TITLE
Incorrect display of BG hero power tooltips 

### ISSUE THREAD

**Issue by piedpiper358** [Zero-to-Heroes/firestone#169]
![2022-08-27_19-31-54](https://user-images.githubusercontent.com/43519401/187095092-c1f76854-1d93-4cbb-93f9-8d87a9aa09b3.png)

![2022-08-27_20-49-22](https://user-images.githubusercontent.com/43519401/187095260-ea18cebc-82e4-42a9-8527-fa8e81c6c526.png)

**Comment by sebastientromp** [comment]
Thanks. I'm actually not sure I generate the right image (with the coin instead of the mana gem) for BG hero powers, so it's something I'll need to fix

**Comment by piedpiper358** [comment]
Yes, definitely! I wanted to report this, but there were more important issues.
But in this case, I'm more about empty purple background.
